src/main/scala/cc/factorie/app/strings/package.scala

```diff
@@ -69,20 +69,69 @@ package object strings {
   val recentYearRegex = "(19|20)\\d\\d".r
   val digitsRegex = "\\d+".r
   val containsDigitRegex = ".*\\d.*".r
-  /** Return input string, with digits replaced, either the whole string with "<YEAR>" or "<NUM>" or just the digits replaced with "#" */
-  def simplifyDigits(word:String): String = {
-    if (recentYearRegex.findFirstIn(word).nonEmpty) "<YEAR>"
-    else if (digitsRegex.findFirstIn(word).nonEmpty) "<NUM>"
-    else if (containsDigitRegex.findFirstIn(word).nonEmpty) word.replaceAll("\\d","#")
-    else word
-  }
   def collapseDigits(word:String): String = {
     if (cc.factorie.app.nlp.lexicon.NumberWords.containsWord(word) || containsDigitRegex.findFirstIn(word).nonEmpty) "0" else word
   }
   def replaceDigits(word:String): String = {
     if (cc.factorie.app.nlp.lexicon.NumberWords.containsWord(word)) "<NUM>" else digitsRegex.replaceAllIn(word, "0")
   }
 
+    /** Return input string, with digits replaced, either the whole string with "<YEAR>" or "<NUM>" or just the digits replaced with "#" */
```

Comment:
I think this overlaps with an earlier version of a PR we recently merged -- verify and remove?

Reply:
verified and removed.